qcodes/instrument_drivers/Keysight/KtM960x.py

```diff
@@ -115,20 +135,21 @@ def _measure(self, key: str = None):
         val_buf = (ctypes.c_double * 1024)()
         actual_size = ctypes.c_int32(0)
         ch_num_buf[0] = 1
-        status = self._dll.KtM960x_MeasurementMeasure(self._session,
-                                                      KTM960X_VAL_MEASUREMENT_TYPE_ALL,
-                                                      1,
-                                                      ch_num_buf,
-                                                      1024,
-                                                      val_buf,
-                                                      ctypes.byref(actual_size)
-                                                      )
+        status = self._dll.KtM960x_MeasurementMeasure(
+            self._session,
+            KTM960X_VAL_MEASUREMENT_TYPE_ALL,
+            1,
+            ch_num_buf,
+            1024,
+            val_buf,
+            ctypes.byref(actual_size)
+        )
 
         if status:
-            print("Driver error! ", status)
-            raise ValueError
+            raise ValueError(f"Driver error: {status}")
         # This might be a bit slow?
-        # Returned as [voltage, current, resistance, status, timestamp, and source]
+        # Returned as [voltage, current, resistance, status,
+        #                                               timestamp, and source]
         v = list(val_buf)[0:actual_size.value]
         val_map = {'voltage': v[0],
```

Comment:
this could be a candidate for the group parameter, or a MultiParameter... not sure)

Reply:
Currently I'm only ever using the 'current' value, so I might make the 'key' a required argument rather than optionally returning that val_map dict..

Reply:
This has been implemented as a multiparameter which returns all values.